knowledge-base/actions/endbug/add-and-commit/action-security.yml

```diff
@@ -0,0 +1,11 @@
+name: Add & Commit
+github-token:
+  action-input:
+    input: token
```

Comment:
@nasablackberry please update `action-input` to github_token, view [here](https://github.com/EndBug/add-and-commit/blob/b3fb60fa0604bb9b1514cb5750dc22e18df06995/action.yml#L64)